### PR TITLE
changelogs missing features after #17589

### PR DESCRIPTION
**Pull request by HeahDude**
| Q | A |
| --- | --- |
| Branch? | master |
| License | MIT |

### CODE REVIEW

**Comment by HeahDude** [comment]
I added a second commit with cs fixes, as it goes in master it should not be a pain to merge older branches as it would to merge it upward, right ?

**Comment by HeahDude** [comment]
Updated here, I've forgotten one method...

Is the second commit needed ? I can remove it. Thanks

**Comment by nicolas-grekas** [comment]
@HeahDude when a CS fix could be applied on lower branches, it must be applied.
Which means you can't trick the process this way :)
I'd recommend you revert the last commit, and *maybe* submit is to the lowest applicable branches.

**Comment by HeahDude** [comment]
@nicolas-grekas thanks, since this fixes are needed for a long time I thought it did not happens because it would have been a pain to merge upward from lower branches.

I revert it right away :)

**Comment by HeahDude** [comment]
Ready!

File: src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+3.1.0
+-----
+
+ * added shortcuts for redirected and forwarded request's profiles in the
```
Comment (xabbuh) [inline src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md]: I am not sure if everyone will understand what is meant with "shortcut".

Comment (HeahDude) [inline src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md]: Right! What about "links" ?

Comment (xabbuh) [inline src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md]: What about "added information about redirected and forwared requests to the profiler"?

Comment (HeahDude) [inline src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md]: Simple and efficient! Thanks :)

**Comment by HeahDude** [comment]
@xabbuh fixed! Thanks.

**Comment by xabbuh** [comment]
:+1:

**Comment by stof** [comment]
> I thought it did not happens because it would have been a pain to merge upward from lower branches.

@HeahDude actually, applying a CS fix only to the latest version would make any future upward merge painful due to conflicts.

**Comment by HeahDude** [comment]
Good to know!

**Comment by fabpot** [comment]
Thank you @HeahDude.

**Pull request merged**